Makefile

```diff
@@ -110,6 +110,9 @@ TRACE_PAR_DIR = trace/par-$(CORE_GENERATOR)-$(CORE_CONFIG)-$(SOC_CONFIG)-$(TECHN
 SIGNOFF_SYN_FORMAL_DIR = check/formal/syn-$(CORE_GENERATOR)-$(CORE_CONFIG)-$(SOC_CONFIG)-$(TECHNOLOGY)-$(MAP_CONFIG)-$(SYN_CONFIG)
 SIGNOFF_SYN_POWER_DIR = check/power/syn-$(CORE_GENERATOR)-$(CORE_CONFIG)-$(SOC_CONFIG)-$(MAP_CONFIG)-$(SYN_CONFIG)
 
+# Define the location of FIRRTL (used by barstools, etc).
+FIRRTL_HOME ?= src/tools/firrtl
```

Comment:
Will this work if it is overridden to point to e.g. the one in rocket-chip? I think its good to allow the user to do that but I just want to make sure that use case is supported.